model/fv_nesting.F90

```diff
@@ -2490,6 +2422,13 @@ subroutine twoway_nest_update(npx, npy, npz, zvir, ncnst, sphum,     &
        first_timestep = .false.
     endif
 
+
+   !!! RENORMALIZATION UPDATE OPTION
+   if (neststruct%nestupdate /= 3 .and. neststruct%nestupdate /= 7 .and. neststruct%nestupdate /= 8) then
+
+   endif
+   !!! END RENORMALIZATION UPDATE
```

Comment:
Note that the "renormalization update" is not used or maintained---this was a fixer to ensure mass conservation on the coarse grid for two-way updating schemes that updated either air mass or tracer mass to the coarse grid. (We don't encourage doing that.)